l10n_it_fatturapa_out/wizard/wizard_export_fatturapa_view.xml

```diff
@@ -6,7 +6,6 @@
         <field name="arch" type="xml">
             <form string="Export Electronic Invoice">
                 <sheet>
-                    <label string="Confirm?"/>
```

Comment:
Replace with an explanatory string, like "in case you want to add to the XML  a PDF version of the invoice, select which report to use"

Reply:
@eLBati done in https://github.com/OCA/l10n-italy/pull/643/commits/4b57d33961e42edb69028474e59c315116ab9663